### ISSUE TITLE
copy-paste error

### ISSUE THREAD

**Issue by AidanRocke** [one-more-minute/Julia-LT#179]
I just installed Juno(Windows 64-bit) and tried copy-pasting code from the startup-tutorial(Tutorial.jl) without success: 

ex. lines 18-20

# Here's a very simple code block:

3 * 6 + 4

note: copy-paste failed between files and within files. 
![juno_cp_failure](https://cloud.githubusercontent.com/assets/5305646/6654376/f95124b4-cab7-11e4-82b2-c50f1980e776.png)

**Comment by one-more-minute** [comment]
So, just to clarify: You select some text, `Ctrl-C`, and trying to paste it somewhere else doesn't do anything?

**Comment by AidanRocke** [comment]
Exactly. Ctrl-V doesn't make a difference and when I right click anywhere on the screen 'paste' is not even an option. 

If you're using windows 64-bit and unable to reproduce the error, we might not be using the same version of Juno.

**Comment by stefanobenati** [comment]
It happens the same to me, with these features too.
1) I am on Windows 7 32 bit,
2) It happens often, but not always;
